src/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why not use [`.env` files](https://vitejs.dev/guide/env-and-mode.html#env-files) for configuration?